spec/unit/sandbox/file_accessor_spec.rb

```diff
@@ -211,6 +211,46 @@ module Pod
         @accessor.license.should == @root + 'LICENSE'
       end
 
+      it 'returns the docs of the specification' do
+        @accessor.docs.should == [
+          @root + 'docs/guide1.md',
+          @root + 'docs/subdir/guide2.md',
+        ]
+      end
+
+      it 'returns the podspecs of the specification' do
+        @accessor.specs.should == [
+          @root + 'BananaLib.podspec',
+        ]
+      end
+
+      it 'returns the matching podspec of the specification' do
+        @accessor.stubs(:specs).returns([@root + 'BananaLib.podspec', @root + 'OtherLib.podspec'])
+        @accessor.send(:podspec_file).should == @root + 'BananaLib.podspec'
+      end
+
+      it 'returns the developer files of the specification' do
+        @accessor.developer_files.should == [
+          @root + 'Banana.modulemap',
+          @root + 'BananaLib.podspec',
+          @root + 'Classes/BananaLib.pch',
```

Comment:
aren't `.pch` files already added in the project references? I checked a project and I see them there, are they being double added now?

Reply:
I thought it was copied to the `Support Files` folder, so editing it would not affect the original file. Did I misunderstand that?

Reply:
Indeed, just double checked and the contents of the file will be appended to `{POD_NAME}-prefix.pch` which is stored in `Pods/Target Support Files/{POD_NAME}-{PLATFORM}/`. Adding it here allows the dev to edit it and run `pod install` to update the generated file